<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="360" />
    <param name="pixel_format" value="yuyv" />
    <param name="color_format" value="yuv422p" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="artag_node" pkg="sarwapangrasa" type="artag_node" output="screen" >
    <rosparam file="$(find sarwapangrasa)/config/camera.yaml" />
    <rosparam file="$(find sarwapangrasa)/config/artag.yaml" />
  </node>
</launch>
